### ISSUE TITLE
Large numbers in y-axis have their first digit cut off

### ISSUE THREAD

**Issue by joaormfsilva** [acl-org/arr-health#21]
As you can see by the attached screenshot, the large numbers in the y-axis have their first digit cut off (e.g. "8,000" instead of "18,000"). Strangely, a period seems to have replaced the digit.

![Screenshot 2024-07-25 at 16-49-47 ARR Dashboard](https://github.com/user-attachments/assets/4cdf1de6-8451-434c-8958-bc0693b701db)

**Comment by joaormfsilva** [comment]
Note: This is browser-dependent. I've just checked that it appears correctly in another (more recent) browser.

**Comment by jkkummerfeld** [comment]
What was the browser that it failed on?

Thanks for reporting this, though it is likely to be something we don't have the time to fix, but we are always happy to receive PRs (which we review and merge).

**Comment by jkkummerfeld** [comment]
Hm, that's not particularly old.

**Comment by joaormfsilva** [comment]
> Hm, that's not particularly old.

No, it's not. The operating system is old-ish (Ubuntu 18.04), but I don't think that should have any impact, as Firefox has updated independently (though not further than 113.0.2, apparently).
Sorry I can't be of more help in troubleshooting this. Hopefully it doesn't seem to be a common problem (from an unscientific, ad-hoc sample of co-workers).

**Comment by jkkummerfeld** [comment]
Still helpful to have the report!
